website/content/en/blog/the-economics-of-the-cncf-platform-engineering-model.md

```diff
@@ -0,0 +1,84 @@
+
+“The desire to refocus delivery teams on their core focus and reduce duplication of effort across the organisation has motivated enterprises to implement platforms for cloud-native computing. By investing in platforms, enterprises can:
+
+1. Reduce the cognitive load on product teams and thereby accelerate product development and delivery
+2. Improve reliability and resiliency of products relying on platform capabilities by dedicating experts to configure and manage them
+3. Accelerate product development and delivery by reusing and sharing platform tools and knowledge across many teams in an enterprise
+4. Reduce risk of security, regulatory and functional issues in products and services by governing platform capabilities and the users, tools and processes surrounding them
+5. Enable cost-effective and productive use of services from public clouds and other managed offerings by enabling delegation of implementations to those providers while maintaining control over user experience”
+
+By analysing this list, we can see the interest of different actors within the enterprise. Reducing the cognitive load of product teams is the direct interest of the members of the product team and the indirect interest of the company as it can accelerate product development. Improving reliability and resilience is the interest of infrastructure teams. Reducing security risks and increasing compliance is the interest of security teams. While cost considerations are a financial topic and depending on the company culture might live with the higher management only. These are the different sides of the market. A successful platform needs to balance the needs of all the sides and orchestrate adoption across all of them.
+
+## Adopting a platform
+
+As already noted, a central question of multi-sided markets is their adoption. The more users are on a platform, the more investment goes into a platform. The more features a platform offers, the more users want to use a platform. But how does one start this cycle? Where is the turning point that will start a self enforcing spiral of adoption?
+
+Platforms require some level of initial coordination. This coordination might come through investment into platform features or internal regulation. The [CNCF Platform Engineering Maturity model](https://tag-app-delivery.cncf.io/whitepapers/platform-eng-maturity-model/) discusses five aspects of maturity: Investment, Adoption, Interfaces, Operations and Measurement. While looking at these five aspects independently is correct from a platform development and maturity assessment perspective, they are interdependent especially early in the lifecycle of a platform.
+
+When a company wants to introduce a platform, I recommend looking at the adoption and investment aspects together. An upfront investment and strong company-wide dedication (executive support) is needed for a successful introduction of a platform.
```

Comment:
I think the WG's maturity model specifically doesn't require executive support up front. There are other ways to evolve a platform.

Reply:
@earth2marsh  It does not require, still it helps a lot. Successful adoptions either have some heroes (who don't mind taking it on them on top of their regular work) or higher-level support.